### PR TITLE
refactor: Update log schema

### PR DESCRIPTION
**Pull request by JordanClemons**
## Description

Update how logging is formatted.

### CODE REVIEW

File: src/library/zoid/modal/component.js [hunk 1]
```diff
@@ -192,10 +192,10 @@ export default createGlobalVariableGetter('__paypal_credit_modal__', () =>
                             index,
                             refIndex,
                             et: 'CLICK',
-                            event_type: 'click',
```
Comment (paypal-rosman) [inline src/library/zoid/modal/component.js:195]: just want to make sure this was a click before and now is modal_rendered?

File: src/library/zoid/modal/component.js [hunk 2]
```diff
@@ -164,9 +164,9 @@ export default createGlobalVariableGetter('__paypal_credit_modal__', () =>
...
-                            link: linkName,
-                            src: src ?? linkName
+                            event_type: 'modal_rendered',
```
Comment (paypal-rosman) [inline src/library/zoid/modal/component.js:167]: just want to make sure this was a click before and now is modal_rendered?

**Review by paypal-rosman — APPROVED** [review]

**Comment by paypal-rosman** [comment]
Approved but have some comments

File: src/library/zoid/modal/component.js (hunk 2 above)
Comment (JordanClemons) [reply src/library/zoid/modal/component.js:167]: Correct

File: src/library/zoid/modal/component.js [hunk 3]
```diff
@@ -192,10 +192,10 @@ export default createGlobalVariableGetter('__paypal_credit_modal__', () =>
...
                             refIndex,
                             et: 'CLICK',
-                            event_type: 'click',
-                            link: 'Calculator',
-                            src: 'Calculator',
-                            amount: value
+                            event_type: 'modal_rendered',
+                            page_view_link_name: 'Calculator',
+                            page_view_link_source: 'Calculator',
+                            calculator_input: value
```
Comment (JustinDoan) [inline src/library/zoid/modal/component.js:198]: Calculator_input is expected as a string in culserv

Comment (JordanClemons) [reply src/library/zoid/modal/component.js:198]: `value` should already be of type string here

**Pull request closed without merging**